ocgis/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: ocgis
+    version: {{environ['CBUILD_OCGIS_VERSION']}}
+
+source:
+    git_url: https://github.com/NCPP/ocgis.git
+    git_tag: {{environ['CBUILD_OCGIS_TAG']}}
+
+build:
+    number: 1
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy
+        - netcdf4
+        - shapely
+        - fiona =={{environ['CBUILD_FIONA_VERSION']}}
```

Comment:
If we can let's avoid the environment variables and the repetition of the run dependencies on the build section.